### DISCUSSION TITLE
I can't get the microphone to work

### DISCUSSION THREAD

**Discussion by Fiddan-code** [mcguirepr89/BirdNET-Pi#1102] [Q&A]
I have installed Birdnet on a Raspberry Pi 4 with 8GB of RAM. Microphone and sound card:
[https://www.amazon.com/dp/B01N905VOY?ref=ppx_yo2ov_dt_b_product_details&th=1]
[https://www.amazon.com/dp/B01AG56HYQ?psc=1&ref=ppx_yo2ov_dt_b_product_details]

I have troubleshooted and reinstalled everything several times and can't get it to work.
Some of the issues, alsamixer doesn't seem to save after you close the terminal, even though I'm using `sudo alsactl store`. When I go back into alsamixer, it still shows 'Card: PulseAudio' even though I've made changes.

I don't know exactly what to specify in the advanced settings for the sound card.
here is what arecord -L gives me
`arecord -L
null
    Discard all samples (playback) or generate zero samples (capture)
default
    Playback/recording through the PulseAudio sound server
lavrate
    Rate Converter Plugin Using Libav/FFmpeg Library
samplerate
    Rate Converter Plugin Using Samplerate Library
speexrate
    Rate Converter Plugin Using Speex Resampler
jack
    JACK Audio Connection Kit
oss
    Open Sound System
pulse
    PulseAudio Sound Server
upmix
    Plugin for channel upmix (4,6,8)
vdownmix
    Plugin for channel downmix (stereo) with a simple spacialization
usbstream:CARD=Headphones
    bcm2835 Headphones
    USB Stream Output
hw:CARD=Device,DEV=0
    USB Audio Device, USB Audio
    Direct hardware device without any conversions
plughw:CARD=Device,DEV=0
    USB Audio Device, USB Audio
    Hardware device with all software conversions
sysdefault:CARD=Device
    USB Audio Device, USB Audio
    Default Audio Device
front:CARD=Device,DEV=0
    USB Audio Device, USB Audio
    Front output / input
dsnoop:CARD=Device,DEV=0
    USB Audio Device, USB Audio
    Direct sample snooping device
usbstream:CARD=Device
    USB Audio Device
    USB Stream Output
usbstream:CARD=vc4hdmi0
    vc4-hdmi-0
    USB Stream Output
usbstream:CARD=vc4hdmi1
    vc4-hdmi-1
    USB Stream Output
`

Plz help :)

**Comment by stevedee** [comment]
From the links, it looks like you have a 3 pole input on USB module and a 4 pole plug on the mic.
If this is not the case, connect mic & module to a laptop/computer and make sure they work.

**Comment by grychofr** [comment]
As stevedee said, you have 3 pole input on USB module and 4 pole plug on the mic, it cannot work. But with your mic, you've also get a 4 to 3 pin stereo adapter. Use it and it should work.
